src/models/s2s.h

```diff
@@ -194,29 +196,56 @@ class DecoderS2S : public DecoderBase {
              && opt<std::string>("original-type") == "nematus")  //
         ("skip", opt<bool>("skip"));
 
+    ABORT_IF(opt<bool>("dec-attention-bilinear-lookup"),
+             "--dec-attention-bilinear-lookup is disabled due to a bug");
```

Comment:
I don't feel comfortable merging this. Either it works, or it should be removed from this PR, unless this is cleared with Marcin.

Reply:
@emjotde should I keep the dot-product attention for the s2s model so that it can be fixed at some point or should I take it out?

As we discussed before, the bug manifests itself in a CUDA kernel, so I would need your help to fix it.